test/cloudtest/test_privatelink_connection.py

```diff
@@ -35,3 +35,31 @@ def test_create_privatelink_connection(mz: MaterializeApplication) -> None:
     )[0][0]
 
     exists(resource=f"vpcendpoint/connection-{connection_id}")
+
+
+def test_create_postgres_privatelink_connection(mz: MaterializeApplication) -> None:
+    mz.environmentd.sql(
+        dedent(
+            """\
+            CREATE CONNECTION kafkaconn TO KAFKA (
+            BROKERS (
+                'customer-hostname-1:9092' USING AWS PRIVATELINK privatelinkconn,
+                'customer-hostname-2:9092' USING AWS PRIVATELINK privatelinkconn (PORT 9093),
```

Comment:
Is it safe to rely on resources created in other tests, especially given that individual tests can be run separately?

This seems to be relying on `test_create_privatelink_connection` being run first, so that the AWS PrivateLink Connection already exists when we create the Kafka Connection using it.

Reply:
Looks like this was combined into one test.